cfgov/v1/atomic_elements/organisms.py

```diff
@@ -611,6 +611,12 @@ class FilterControls(BaseExpandable):
                                      label='Filter Date Range')
     output_5050 = blocks.BooleanBlock(default=False, required=False,
                                       label="Render preview items as 50-50s")
+    should_link_image = blocks.BooleanBlock(
+        default=False,
+        required=False,
+        help_text=('Check this to link all images to the URL of the first '
+                   'link in their unit\'s list, if there is a link.')
```

Comment:
I think this text is borrowed from a previous place we implemented this (25/75 or 50/50 Image and Text), and I think I suggested it there. I wanted it to be clear that if the item has multiple links, it will link the image to the first one. That said, this is a non-issue for this module, where it will only ever link to the page for the filtered item. I think your suggestion is good.